Apps/Medication/src/Controllers/MedicationStatementController.cs

```diff
@@ -75,6 +75,23 @@ public MedicationStatementController(IAuthorizationService authorizationService,
         [Route("{hdid}")]
         [Authorize(Policy = "PatientOnly")]
         public async Task<ActionResult> GetMedicationStatements(string hdid)
+        {
+            return await this.GetMedicationStatements(hdid, null).ConfigureAwait(true);
+        }
+
+        /// <summary>
+        /// Gets a json of medication record.
+        /// </summary>
+        /// <returns>The medication statement records.</returns>
+        /// <param name="hdid">The patient hdid.</param>
+        /// <param name="protectiveWord">The clients protective word for Pharmanet.</param>
+        /// <response code="200">Returns the medication statement bundle.</response>
+        /// <response code="401">The client is not authorized to retrieve the record.</response>
+        [HttpGet]
+        [Produces("application/json")]
+        [Route("{hdid}")]
```

Comment:
Do we need to map the protectiveWord into the route?

Reply:
We’re going to add the protective word to the header.